training-service/src/app/content/course/component/course-list/course-list.component.html

```diff
@@ -5,6 +5,10 @@
         <input type="checkbox">
           <span>{{ item.name }}</span>
       </label>
+      <span>
+        opis: {{ item.description }}, czas trwania: {{ item.duration * 1000 | date:'mm:ss' }} minut,
```

Comment:
please extract item.duration * 1000 to a function and do not do math operations in template.

Reply:
done